ckan/lib/search/common.py

```diff
@@ -77,11 +79,13 @@ def make_connection(decode_dates=True):
                                        quote_plus(solr_password),
                                        solr_url)
 
+    timeout = tk.asint(tk.config.get('solr_timeout', 60))
```

Comment:
For core code is better to use core imports, in this case `from ckan.common import config, asint` and